client/app/puregetters/requests.coffee

```diff
@@ -0,0 +1,106 @@
+{RequestStatus, Requests} = require '../constants/app_constants'
+
+module.exports =
+    getRequests: (state) ->
+        state.get('requests')
+
+    get: (state, req) ->
+        @getRequests(state).get req
+
+    isLoading: (state, req) ->
+        RequestStatus.INFLIGHT is @get(state, req)?.status
+
+    getRefreshes: (state) ->
+        state.get('refreshes')
+
+    isRefreshError: (state) ->
+        @getRefreshes(state).get('errors')?.length
+
+    isRefreshing: (state)->
+        @getRefreshes(state).size isnt 0 or
+        @isLoading(state, Requests.REFRESH_MAILBOX)
+
+    isIndexing: (state, accountID) ->
```

Comment:
Not sure if these are still relevant. These is mostly translated code from previous state. Let's agree on naming, events and expected behaviours tomorrow and then we can remove what will not be used.